src/modules/poweraccent/PowerAccentKeyboardService/KeyboardListener.cpp

```diff
@@ -115,6 +115,13 @@ namespace winrt::PowerToys::PowerAccentKeyboardService::implementation
         {
             Logger::info(L"Show toolbar. Letter: %d, Trigger: %d", letterPressed, triggerPressed);
 
+            if (((GetAsyncKeyState((int)letterPressed) & 0x8000) == 0))
+            {
+                triggerPressed = 0;
```

Comment:
The check is more tightly coupled with `if` statement at line #105.
Would be great to add
`const bool letterIsStillPressed = (GetAsyncKeyState((int)letterPressed) & 0x8000) == 0;` and check it in the if statement at line #105
Thanks!

Reply:
Agree, thx!